tests/db/test_bigchain_api.py

```diff
@@ -989,41 +989,51 @@ def test_get_owned_ids_calls_get_outputs_filtered():
     assert res == gof()
 
 
+@pytest.mark.tendermint
```

Comment:
I don't know if we need to mark them for tendermint yet. Maybe fixing them but still skipping them is better, @kansi ?

Reply:
And some parts of this file are also changed by @z-bowen s PR: https://github.com/bigchaindb/bigchaindb/pull/2366/files#diff-5dbcdadd05006f510ee75ac7dae5db47

Reply:
It does not make sense to change tests that are not executed. Also, if the tests have value, why disable them?

Reply:
As far as I know, making tests 'work again' was supposed to be an extra step. I don't mind if we change them in here and make them work.

Reply:
I concur with @ldmberman , we have been modifying tests in other PRs which aren't enabled. That doesn't give any insight into whether the test actually passes or not. Also, I really don't mind having more tests :smile: